angular: broken environment

i have encountered this error on 0534a403aac5eb28109574d62ff8a7ef5724dbc1:
```
✘ [ERROR] NG9: Property 'mediaUrl' does not exist on type '{ production: boolean; apiUrl: string; }'. [plugin angular-compiler]

    src/app/components/event-list/event-list.component.html:5:44:
      5 │ ...="event.picture" src="{{env.mediaUrl}}{{event.picture}}" alt="Pi...
```
This is caused by the absence of `mediaUrl` in `environment.development.ts`.

--- a/web_frontend/src/environments/environment.development.ts
+++ b/web_frontend/src/environments/environment.development.ts
@@ -1,4 +1,5 @@
 export const environment = {
   production: false,
   apiUrl: 'http://localhost:8000/',
+  mediaUrl: 'http://localhost:8000/',
 };

